google/cloud/aiplatform/featurestore/entity_type.py

```diff
@@ -37,7 +37,7 @@
 
 from google.cloud import bigquery
 
-_LOGGER = base.Logger(__name__)
+_LOGGER = base.Logger()
```

Comment:
This changes the behavior. Previously every module had it's own logger (based on `__name__`).
Not sure what's the best way.

Reply:
Nope, that's actually still the behavior! __name__ is the default name in the logger constructor.

This 1. removes boilerplate and 2. prevents bad patterns (__name__ should be used whenever possible, this makes it harder to make a mistake)

Reply:
>Nope, that's actually still the behavior! name is the default name in the logger constructor.

I'm not sure this is the case. Previously, every python module passed `__name__` to the `Logger` class constructor. So logger for every module was getting a different name.
Now, the `__name__` is only used in the `logger` module. So all logger names will be the same now.
The key piece of information is that `__name__` is bound to the module where it's used.

Reply:
+1, when `_LOGGER = base.Logger()` is being called in different modules, the default value for `__name__` is still the base, though, it does not seem we are enforcing any logging formatting to show the `__name__`. Prefer to be able to trace the logger since we can.